modules/host/negotiate.go

```diff
@@ -234,7 +228,7 @@ func (h *Host) rpcContract(conn net.Conn) (err error) {
 	}
 
 	// Add the collateral to the transaction, but do not sign the transaction.
-	collateralTxn, txnID, err := h.addCollateral(unsignedTxn, terms)
+	collateralTxn, txnBuilder, err := h.addCollateral(unsignedTxn, terms)
```

Comment:
I agree, but the host and renter files are going to be changing dramatically enough that I'm not really worried about maintaining them.